packages/next-swc/crates/next-core/src/web_entry_source.rs

```diff
@@ -105,43 +104,50 @@ pub async fn get_web_runtime_entries(
     };
 
     runtime_entries.push(
-        RuntimeEntry::Source(SourceAssetVc::new(next_js_file_path("dev/bootstrap.ts")).into())
-            .cell(),
+        RuntimeEntry::Request(
```

Comment:
I can't seem to remember, it's been a while since I touched this. Might have been to expose it under a particular path? In any case, it's not relevant to production, so I'll undo this.